store/tikv/2pc.go

```diff
@@ -1072,6 +1141,11 @@ func (c *twoPhaseCommitter) execute(ctx context.Context) (err error) {
 	}
 	c.commitTS = commitTS
 
+	if c.isOnePC() {
+		c.txn.commitTS = c.commitTS
+		return nil
```

Comment:
Need we close `ttlManager`?

Reply:
The ttlManager is closed [here](https://github.com/pingcap/tidb/blob/2fa52ee0328c2935d1bb90a128364cc531c5b680/store/tikv/txn.go#L248) . The ttlManager should last until the primary committed, so for 1PC, it's ok to close it just after the `Commit` function returns. It's special for async commit because it's primary is committed asynchronously, if my understanding is correct. Also PTAL @cfzjywxk

Reply:
Ah, I see. Never mind.